src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+export default function App() {
+  return (
+    <div className="container">
```

Comment:
- App component should implement render function, which displays Display and ButtonPanel components  see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/Display&Button.png)

Reply:
Done!